pkgs/development/haskell-modules/hackage-packages.nix

```diff
@@ -143332,7 +143373,7 @@ self: {
 
   "insert-ordered-containers" = callPackage
     ({ mkDerivation, aeson, base, base-compat, hashable, lens
-     , optics-core, optics-extra, QuickCheck, semigroupoids, semigroups
```

Comment:
You must not override anything manually in `hackage-packages.nix`. This file gets regenerated every night and will override your changes.
You should set all version overrides in `pkgs/development/haskell-modules/configuration-common.nix`.